InnerEye/ML/baselines_util.py

```diff
@@ -203,6 +205,7 @@ def get_comparison_baselines(outputs_folder: Path, azure_config: AzureConfig,
                 # noinspection PyUnresolvedReferences
                 if path is not None and not path.exists():
                     logging.warning("    ... but it does not exist")
```

Comment:
this whole block from line 202 onwards, not sure this adds much value.  Do we not yet have all the information in the logs, from line 191 and 179? Also, in which cases is `download__outputs_from_run` going to return something that is not None, but the file does not actually exist? This would simplify the checks.